blocks/article-list/article-list.js

```diff
@@ -8,7 +8,15 @@ import { createCardBlock } from '../card/card.js';
 export default async function decorate(block) {
   const filters = removeEmptyKeyOrValue(readBlockConfig(block));
   const isEmptyFilter = Object.keys(filters).length === 0;
-  if (isEmptyFilter && document.location.pathname.startsWith('/author/')) {
+  if (isEmptyFilter && document.location.pathname.startsWith('/category/')) {
+    // auto-detect category, e.g. https://www.24life.com/category/lifestyle
+    const parts = document.location.pathname.split('/');
+    const askedCategory = parts[parts.length - 1];
+    filters.categories = askedCategory.replace(/-/g, ' ');
+    const firstContainer = document.createElement('div');
+    firstContainer.innerHTML = `<h1>IN: ${filters.categories.toUpperCase()}</h1>`;
```

Comment:
This text is displaying twice.
The text reads "BY: IN: LIFESYTLE", it should be "IN: LIFESTYLE". It does this via the Author templates' author.css. Just remove the metadata block from default.docx.

Reply:
yes @rrusher  - I did remove the metadata block from default but still we are seeing that - https://issue37-foldermapping-v1--24life--hlxsites.hlx.live/category/lifestyle . This issue is happening only on this page. Rest all the categories are popping up fine - 

https://issue37-foldermapping-v1--24life--hlxsites.hlx.live/category/Discover
https://issue37-foldermapping-v1--24life--hlxsites.hlx.live/category/Fashion
https://issue37-foldermapping-v1--24life--hlxsites.hlx.live/category/Flexibility
https://issue37-foldermapping-v1--24life--hlxsites.hlx.live/category/Game-Changers
https://issue37-foldermapping-v1--24life--hlxsites.hlx.live/category/Motivate
https://issue37-foldermapping-v1--24life--hlxsites.hlx.live/category/Motoring

Seems like some issue with the caching at the CDN, may be. Please advise.